src/backend/renderer/gles2/mod.rs

```diff
@@ -514,6 +517,7 @@ impl From<Gles2Error> for SwapBuffersError {
             | x @ Gles2Error::BufferAccessError(_)
             | x @ Gles2Error::MappingError
             | x @ Gles2Error::UnexpectedSize
+            | x @ Gles2Error::CreateProgram
```

Comment:
Please note, that I don't know if this error should be put here or to `SwapBuffersError::ContextLost`.

Reply:
This one will fail initializing, which is not temporary. So context lost please. :)

Reply:
okeydokey!

Reply:
done!